tencentcloud/data_source_tc_cam_group_policy_attachments.go

```diff
@@ -49,7 +49,7 @@ func dataSourceTencentCloudCamGroupPolicyAttachments() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				ValidateFunc: validateAllowedStringValue(CAM_POLICY_CREATE_STRATEGY),
-				Description:  "Type of the policy strategy. 'User' means customer strategy and 'QCS' means preset strategy.",
+				Description:  "Type of the policy strategy. Valid values are 'User', 'QCS', '', 'User' means customer strategy and 'QCS' means preset strategy.",
```

Comment:
1. Valid values are 'User', 'QCS' and '' (empty)
2. '' (empty) means _____________